### DISCUSSION TITLE
Request to /auth/forgot-password returns 400

### DISCUSSION THREAD

**Discussion by matrixersp** [strapi/strapi#7601] [Legacy Topics]
When I make a request to /auth/forgot-password or /admin/auth/forgot-password I get a 400 error, I'm using Strapi v3.0.6 
This is the result I get in Postman:
```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": {},
    "data": {}
}
```
And this is the error I get in Strapi admin panel:
![Screenshot from 2020-08-25 21-37-33](https://user-images.githubusercontent.com/19875946/91225671-e88e6200-e71b-11ea-8335-b0a9ad1fd6a9.png)

**Comment by derrickmehaffy** [comment]
Which email provider are you using?

**Reply by matrixersp** [reply]
Gmail

**Reply by derrickmehaffy** [reply]
I mean within Strapi, as Strapi has to send an email. Do you have a mail provider configured or are you just using the default?
Also I wasn't able to reproduce this on the current release (v3.1.4)

If you are using the default and running locally, most likely your ISP has blocked outbound port 25 email, it's quite common.

**Reply by matrixersp** [reply]
Yes I'm using the default

**Reply by derrickmehaffy** [reply]
Yeah in that case, it's most likely your ISP blocking the outbound SMTP request on port 25

**Reply by matrixersp** [reply]
Yes right, it's as you said the problem is with my ISP. Thank you!
